src/dev/flang/be/jvm/classfile/Expr.java

```diff
@@ -92,6 +174,200 @@ public void code(ClassFile.ByteCodeWriter ba, ClassFile cf)
     {
       ba.write(_bc);
     }
+
+    @Override
+    public void buildStackMapTable(StackMapTable smt, Stack<VerificationType> stack, List<VerificationType> locals)
+    {
+      if (_bc == BC_RETURN ||
```

Comment:
I think it would be better to do a `switch (_bc[0]) { case O_return, O_ireturn, ... -> ` here.